tsconfig.build.json

```diff
@@ -1,5 +1,4 @@
 {
-	"extends": "./tsconfig.json",
-	"include": ["src"],
-	"exclude": ["node_modules", "test"]
-}
\ No newline at end of file
+  "extends": "./tsconfig.json",
+  "include": ["src"]
+}
```

Comment:
Why has the `exclude` property been removed?

Reply:
I understand that `include` is enough if we want to build only `src` folder, using both doesn't change anything and I find it even more confusing, as exclude didn't mention all other folders in the repo anyway.